packages/web-components/fast-foundation/src/select/select.ts

```diff
@@ -305,6 +306,25 @@ export class Select extends FormAssociatedSelect {
         this.updateValue();
     }
 
+    protected setDefaultSelectedOption(): void {
+        const options: ListboxOption[] =
+            this.options ?? Array.from(this.children).filter(Listbox.slottedOptionFilter);
+
+        const selectedIndex = options?.findIndex(
+            el =>
+                el.getAttribute("selected") !== null ||
```

Comment:
My bad, there's a better check for this:

```suggestion
                el.hasAttribute("selected") ||
```